common/src/main/scala/com/gu/recipeasy/db/DB.scala

```diff
@@ -189,17 +189,43 @@ class DB(contextWrapper: ContextWrapper) {
   }
 
   def progressBarRatio(): Double = {
-    val Key = "verificationCompletionRatio"
 
+    // progressBarRatio ratio is a number between 0 and 1
+
+    def pbr(): java.lang.Double = {
```

Comment:
You are right. I have this habit to comment a lot, but admittedly I got it as a protection against evilness while using untyped languages :)